src/uct/api/uct.h

```diff
@@ -1329,6 +1380,28 @@ ucs_status_t uct_md_config_read(const char *name, const char *env_prefix,
                                 uct_md_config_t **config_p);
 
 
+
+/**
+ * @ingroup UCT_MD
+ * @brief Check if remote sock address is reachable from the memory domain.
+ *
+ * This function checks if a remote sock address can be reached from a local
```

Comment:
local memory domain which is trying to establish connection to the remote peer's sockaddr. 
reg. ep create, yes. a new function - uct_ep_create_sockaddr() which will be called on the client side and create a connection - an endpoint, to the remote peer's sockaddr - to the server.